x-pack/plugins/endpoint/public/embeddables/resolver/view/defs.tsx

```diff
@@ -0,0 +1,173 @@
+            <path
+              strokeWidth="3.00000004"
+              stroke={NamedColors.ok}
+              strokeLinecap="butt"
+              strokeLinejoin="round"
+              strokeOpacity="0.94117647"
+              d="M 3,1014.0043 C 1.9285712,955.24249 2.5,954.53017 50,955.42048 h 300 c 47.14282,-1.33548 47.71427,-0.62325 47,58.58382"
+            />
+            <path
+              strokeWidth="3.00000004"
+              stroke={NamedColors.ok}
+              strokeLinecap="butt"
+              strokeLinejoin="round"
+              strokeOpacity="0.94117647"
+              d="m 397,1002.3622 c 1.07142,47.1428 0.5,47.7143 -47,47 H 50 c -47.1428203,1.0714 -47.71427,0.4994 -47,-47"
+            />
+          </g>
+        </g>
+      </g>
+    </symbol>
+    <symbol
+      id={SymbolIds.processNode}
+      viewBox="-10 0 128.6889 29.31389"
+      preserveAspectRatio="xMidYMid meet"
+    >
+      <path
+        d="M 1.5453386,0.77661425 H 107.34961 c 0.37098,0 0.66964,0.20676641 0.66964,0.46360185 V 28.180641 c 0,0.256835 -0.29866,0.463601 -0.66964,0.463601 H 1.5453386 c -0.3709839,0 -0.66964599,-0.206766 -0.66964599,-0.463601 V 1.2402161 c 0,-0.25683544 0.29866209,-0.46360185 0.66964599,-0.46360185 z"
+        fill={`url(#${PaintServerIds.darkLinearReflect})`}
+        fillOpacity="1"
+        stroke={NamedColors.ok}
+        strokeWidth="0.889"
+        strokeMiterlimit="4"
+        strokeOpacity="1"
+        paintOrder="normal"
+      />
+    </symbol>
+    <symbol
+      id={SymbolIds.processNodeWithHorizontalRule}
+      viewBox="-10 0 128.6889 29.31389"
+      preserveAspectRatio="xMidYMid meet"
+    >
+      <path
+        d="M 1.5453386,0.77661425 H 107.34961 c 0.37098,0 0.66964,0.20676641 0.66964,0.46360185 V 28.180641 c 0,0.256835 -0.29866,0.463601 -0.66964,0.463601 H 1.5453386 c -0.3709839,0 -0.66964599,-0.206766 -0.66964599,-0.463601 V 1.2402161 c 0,-0.25683544 0.29866209,-0.46360185 0.66964599,-0.46360185 z"
+        fill={`url(#${PaintServerIds.darkLinearReflect})`}
+        fillOpacity="1"
+        stroke={NamedColors.ok}
+        strokeWidth="0.889"
+        strokeMiterlimit="4"
+        strokeOpacity="1"
+        paintOrder="normal"
+      />
+      <line x1="1.425" x2="108.5" y1="10" y2="10" stroke={NamedColors.ok} strokeWidth="0.449" />
+    </symbol>
+
+    <symbol id={SymbolIds.solidHexagon} viewBox="0 0 200 200" preserveAspectRatio="xMidYMid meet">
+      <g transform="translate(0,-97)">
+        <path
+          transform="matrix(1.6461 0 0 1.6596 -56.401 -64.183)"
+          d="m95.148 97.617 28.238 16.221 23.609 13.713 0.071 32.566-0.071 27.302-28.167 16.344-23.68 13.59-28.238-16.221-23.609-13.713-0.07098-32.566 0.07098-27.302 28.167-16.344z"
+          fill="inherit"
+          strokeWidth="15"
+          stroke="inherit"
+        />
+      </g>
+    </symbol>
+  </>
+));
+
+export const SymbolDefinitions = memo(() => (
+  <svg className="resolver_defs">
```

Comment:
This classname seems unused

Reply:
It can be removed. At some point I might need some advice on turning this into a sort of singleton component (which I think you suggested a while back) that renders in the "outermost"  Resolver, but not the inner ones.